html/semantics/forms/the-form-element/form-requestsubmit.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<body>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<iframe name="iframe" src="about:blank"></iframe>
+
+<script>
+test(() => {
+  document.body.insertAdjacentHTML('afterbegin', '<form><input type="reset"></form>');
```

Comment:
Also test button element with type=button and type=reset

Reply:
Done.